lib/modules/ens/index.js

```diff
@@ -25,6 +25,38 @@ class ENS {
     this.addENSToEmbarkJS();
     this.configureContracts();
     this.registerEvents();
+    this.registerConsoleCommands();
+  }
+  
+
+  registerConsoleCommands() {
+    this.embark.registerConsoleCommand((cmd, _options) => {
+      let cmdName = cmd.split(' ')[0];
+      let domain = cmd.split(' ')[1];
+      return {
+        match: () => cmdName === 'resolve',
+        process: () => global.EmbarkJS.Names.resolve(domain).then(this.logger.info)
```

Comment:
process takes a callback as a params with the following parameters:
`callback(error | null, result: string)`
I think that using the logger may break the output and this is how we let the repl return correctly

Reply:
@alaibe fixed & rebased